examples/games/game_menu.rs

```diff
@@ -45,10 +45,62 @@ fn setup(mut commands: Commands) {
     commands.spawn(Camera2dBundle::default());
 }
 
+// This outermost node encompasses the entire Window (100% width and height)
+// and centers all content vertically (align_items) and horizontally (justify_content)
+fn outer_node(
+    commands: &mut Commands,
+    marker: impl Component,
+    spawn_children: impl FnOnce(&mut ChildBuilder),
+) {
+    commands
+        .spawn((
+            NodeBundle {
+                style: Style {
+                    align_items: AlignItems::Center,
+                    justify_content: JustifyContent::Center,
+                    width: Val::Percent(100.0),
+                    height: Val::Percent(100.0),
+                    ..default()
+                },
+                ..default()
+            },
+            marker,
+        ))
+        .with_children(spawn_children);
+}
+
+// This node is always a direct child of the `outer` node, and arranges contents in a column, from top to bottom
+fn inner_node<T: Into<BackgroundColor>>(
```

Comment:
This pattern was previously used on every single screen except for the Splash screen, and so was repeated in five places